templates/base.html

```diff
@@ -18,6 +18,7 @@
     <!-- Import the custom Bootstrap 4 theme from our hosted CDN -->
     <link rel="stylesheet" href="https://demo.productionready.io/main.css" />
 
+    <meta name="turbo-cache-control" content="no-preview" />
```

Comment:
Disabling previews since it causes flashing when revisiting article lists you've seen before.